dasharo-security/tpm2-commands.robot

```diff
@@ -312,6 +313,43 @@ TPMCMD014.001 Encrypt and Decrypt non-rootfs partition (Ubuntu 22.04)
     Execute Command In Terminal    rm -f key seal.* prim.* test-partition
     Execute Linux Tpm2 Tools Command    tpm2_evictcontrol -c 0x81010001
 
+TPMCMD015.001 Encrypt and Decrypt rootfs partition (Ubuntu 22.04)
+    [Documentation]    Test encrypting and decrypting rootfs partition using
+    ...    TPM. This test assumes that Ubuntu already has encrypted rootfs. This
+    ...    test is executed if rootfs_encryption_test variable is True (can be
+    ...    set via command line: -v rootfs_encryption_test:True).
+    # 1. Add boot option for Ubuntu with ecrypted rootfs:
+    Power On
+    Enter Setup Menu Tianocore
+    Add Boot Option    ubuntu     ubuntu-enc    ubuntu-enc-rootfs
+    Save Changes And Reset    2    2
+
+    # 2. Boot to ubuntu with encrypted rootfs:
+    Boot System Or From Connected Disk    ubuntu-enc-rootfs
+    Unlock Rootfs
```

Comment:
@DaniilKl where is definition of this KW?

Reply:
Added here d87d7ccf465aeb884c527bd90a7be66a20c784c5